subprojects/docs/src/snippets/buildCache/cacheable-bundle/groovy/build.gradle

```diff
@@ -8,10 +8,16 @@ class NpmTask extends DefaultTask {
     @Input
     final ListProperty<String> args = project.objects.listProperty(String)
 
+    @InputDirectory
```

Comment:
Fedor, please watch this discussion on what a better fix for this will look like: https://youtu.be/dDH3rNwMTQ0?t=2959

Basically, we want to avoid assigning actual values or conventions eagerly during task initialization, as we have no control over order of evaluation.

Reply:
Thanks for the link! Fixed.